doc/source/user_guide/training.rst

```diff
@@ -45,9 +44,6 @@ Train on prediction data
 
      td.add_to_project(project)
 
-Once you have training data in your project, you can use the web app to
-train a model.
-
```

Comment:
Maybe this:
Your training data is imported and assigned to a project. (_optional:_ You can now configure your AI model.)
If the _optional_ next step used, an hyperlink to the model config api reference is to be added at the end of this page.